src/CTe/DacteV3.php

```diff
@@ -1746,13 +1746,14 @@ protected function zDescricaoCarga($x = 0, $y = 0)
         $x = $oldX;
         $this->pdf->Line($x, $y, $w + 1, $y);
         //Identifica código da unidade
-        //01 = KG (QUILOS)
-        if ($this->getTagValue($this->infQ->item(0), "cUnid") == '01') {
-            $qCarga = $this->getTagValue($this->infQ->item(0), "qCarga");
-        } elseif ($this->getTagValue($this->infQ->item(1), "cUnid") == '01') {
-            $qCarga = $this->getTagValue($this->infQ->item(1), "qCarga");
-        } elseif ($this->getTagValue($this->infQ->item(2), "cUnid") == '01') {
-            $qCarga = $this->getTagValue($this->infQ->item(2), "qCarga");
+        //01 = KG (QUILOS) ou 02 = (TONELADAS)
+        foreach ($this->infQ as $infQ) {
+            if ($this->getTagValue($infQ, "cUnid") == '01') {
+                $qCarga = $this->getTagValue($infQ, "qCarga");
```

Comment:
nesse FOR deveria somar as quantidades de cada qCarga para ser exibido na linha 1764 correto?